Cargo.toml

```diff
@@ -15,7 +15,7 @@ syn = "0.11.11"
 heck = "0.3.0"
 
 [dev-dependencies]
-diesel = {version = "1", features = ["postgres"] }
+diesel = {version = ">=1.1", features = ["postgres"] }
```

Comment:
This means that 2.0 would be valid too, just put "1.1" instead

Reply:
@Eijebong Is there no way to say "support every `1.x` version after `1.1`?

Reply:
Well "1.1" is equivalent to "^1.1" which means, any version greater than 1.1 that's semver compatible with it.

Reply:
@Eijebong Ah, I would have thought `1.1` means every `1.1.x` version.

Reply:
Nah that would be "~1.1" IIRC